telemetry/definitions/commonDefinitions.json

```
@@ -354,6 +354,16 @@
             "type": "string",
             "description": "The id of the experiment being activated or deactivated"
         },
+        {
+            "name": "settingId",
+            "type": "string",
+            "description": "The id of the setting being changed"
```

I think we should give an example in the description:
```suggestion
            "description": "The id of the setting being changed, e.g. codewhisperer_autoSuggestionActivation"
```